x-pack/plugins/triggers_actions_ui/public/application/lib/get_defaults_for_action_params.ts

```diff
@@ -38,5 +39,59 @@ export const getDefaultsForActionParams = (
         spaceId: `{{${AlertProvidedActionVariables.ruleSpaceId}}}`,
       };
       return xmattersDefaults;
+    case '.torq':
+      const torqDefaults = {
+        body: JSON.stringify(
```

Comment:
why we are the only one who use  JSON.stringify?

Reply:
you can export this thing to another file and return it here

Reply:
Don't you already know we're special?